node/cmd/ccq/utils.go

```diff
@@ -197,3 +221,42 @@ func validateRequest(logger *zap.Logger, perms Permissions, apiKey string, qr *g
 
 	return nil
 }
+
+const (
+	GuardianKeyArmoredBlock = "WORMHOLE GUARDIAN PRIVATE KEY"
+)
+
+// loadSignerKey loads the signer key from disk.
+func loadSignerKey(filename string) (*ecdsa.PrivateKey, error) {
+	f, err := os.Open(filename)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open file: %w", err)
+	}
+
+	p, err := armor.Decode(f)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read armored file: %w", err)
+	}
+
+	if p.Type != GuardianKeyArmoredBlock {
+		return nil, fmt.Errorf("invalid block type: %s", p.Type)
```

Comment:
It might be nice (for the SRE / ops folks here) to put what is being expected in this error message. What do you think?
```suggestion
		return nil, fmt.Errorf("invalid block type: '%s', expecting '%s'", p.Type, GuardianKeyArmoredBlock)
```

Reply:
This code has been ripped out in favor of the recently added library function.